controller/build/build.go

```diff
@@ -99,36 +99,52 @@ func RunBuild(ctx context.Context, dockerCli command.Cli, in controllerapi.Build
 		return nil, nil, err
 	}
 	if in.ExportPush {
-		if in.ExportLoad {
-			return nil, nil, errors.Errorf("push and load may not be set together at the moment")
-		}
 		if len(outputs) == 0 {
```

Comment:
This case is not needed anymore. Else case behaves same way if `len(outputs) == 0`. Same for `docker`.

Reply:
Ah right, we are already checking if push/load is already used.